codeforboston/maple
# 1208 make settings a hyperlink on profile page

# Summary

Due to the 'Make public/private' button already in place, changing the wording of the message to reflect the page properly.

# Screenshots

![image](https://github.com/codeforboston/maple/assets/68314210/573b3a7f-b375-4671-9723-3072a6a4327c)

# Steps to test/reproduce

1. If not logged in, Login to your account.
2. Click on your account name at the top right and click on view profile.
3. Make your account public and/or private to confirm changes are correct on both.
